server/src/main/java/org/elasticsearch/index/mapper/GeoPointScriptFieldType.java

```diff
@@ -46,34 +44,10 @@
             lookup,
             ctx
         ) {
-        private final GeoPoint scratch = new GeoPoint();
 
         @Override
         public void execute() {
-            try {
-                Object value = XContentMapValues.extractValue(field, leafSearchLookup.source().source());
```

Comment:
I did not realize that the extraction from source is different for geo_point fields. That is why tests are failing. Why is that? Also it seems like in this case we don't support multiple values, but one geo_point, which is made of two values, correct?

Reply:
It's because `XContentMapValues.extractRawValues()` automatically unwraps arrays, but `geo_point` can accept array-formatted input.  I think we should still be able to support multiple inputs though, so if we don't that's a bug.